src/cacheout/cache.py

```diff
@@ -442,6 +442,29 @@ def expire_times(self) -> t.Dict[t.Hashable, T_TTL]:
         """
         with self._lock:
             return self._expire_times.copy()
+        
+    def item_ttl(self, key: t.Hashable) -> T_TTL:
+        """
+        Returns the remaining time to live of a key that has a TTL.
+
+        Args:
+            key: Cache key to review.
+
+        Returns:
+            float: the remaining time to live of item
+            int: ``-1`` if the key exists but has no associated expire, ``-2`` if key didn't exist.
+        """
+        with self._lock:
+            expire_time = self._expire_times.copy().get(key)
+            if not expire_time:
+                if not self._get(key):
```

Comment:
Key values can be empty (e.g. booleans) and `self._get(key)` will set the key value to the cache-level default if it was set (e.g. `cache = Cache(..., default="something")`.

So should use `self._has(key)` to check for presence instead.